docs/src/components/about/OurValues.tsx

```diff
@@ -14,23 +14,23 @@ import SectionHeadline from 'docs/src/components/typography/SectionHeadline';
 const values = [
   {
     title: 'Put community first 💙',
-    description: 'We never lose sight of who we’re serving and why.',
+    description: "We never lose sight of who we're serving and why.",
     lightIcon: 'url(/static/branding/about/illustrations/community-light.svg)',
     darkIcon: 'url(/static/branding/about/illustrations/community-dark.svg)',
     width: 92,
     height: 84,
   },
   {
     title: 'Avoid bureaucracy 🚫',
-    description: 'We’re so not corporate — and we like it that way.',
+    description: 'Were so not corporate — and we like it that way.',
```

Comment:
Just a quick note that removing the apostrophe here is wrong 😬 
It's supposed to be "**We're**" (_We are_)

Reply:
Oops, fixed in HEAD.